litrpc/lit-accounts.proto

```diff
@@ -148,6 +154,28 @@ message UpdateAccountRequest {
     string label = 4;
 }
 
+message UpdateAccountBalanceRequest {
```

Comment:
so expanding on my other comment: it doesnt really make sense to starting off the PR with rpc definitions as we dont really know how this is going to be used yet etc.

Reply:
Addressed in the new PR :)